main.js

```diff
@@ -0,0 +1,109 @@
+var removeLinkClass = 'remove-link';
+var commentsLinkClass = 'comments-link';
+var commentsIdCounter = 1;
+
+
+var clicked = function () {    
+    var userName = $('#name').val();
+    var post = $('#message').val();     
+    
+    
+    //Post variables for new HTML nodes
+    var postsDiv = document.querySelector('.posts');
+    var newPostDiv = document.createElement('div');    
+    var newPostP = document.createElement('p');
+    var newPostSpan = document.createElement('span');      
+    var newPostSpan2 = document.createElement('span');   
+    
+    
+    //Assigning class names to post classes
+    newPostSpan.className = removeLinkClass;
+    newPostSpan2.className = commentsLinkClass;
+    
+    //Post Variables for new text and links
+    var classOneTextNode = document.createTextNode('Remove ');
+    var classTwoTextNode = document.createTextNode('Comments ');
+    var newPostTextNode = document.createTextNode(post + ' - Posted By: ' + userName);    
+    
+    //Put text into span(s)
+    newPostSpan.appendChild(classOneTextNode);
+    newPostSpan2.appendChild(classTwoTextNode);     
+
+    //Put <span> (s) into <p>
+    newPostP.appendChild(newPostSpan);
+    newPostP.appendChild(newPostSpan2);    
+    newPostP.appendChild(newPostTextNode);    
```

Comment:
This is a bit of a convoluted way to do this - it would probably be more readable to use a string of HTML to accomplish this, though I like the ingenuity here.